assets/src/components/mapMarkers.tsx

```diff
@@ -409,3 +412,26 @@ export const GarageMarkers = ({ zoomLevel }: { zoomLevel: number }) => (
     ))}
   </>
 )
+
+const locationLeafletIcon = (selected?: boolean): Leaflet.DivIcon =>
+  Leaflet.divIcon({
+    html: locationDotIcon,
+    className:
+      "c-location-dot-icon" +
+      (selected ? " c-location-dot-icon--selected" : ""),
+    iconAnchor: new Leaflet.Point(9, 24),
+    iconSize: [24, 18],
+  })
+
+export const LocationMarker = ({
+  location,
+  selected,
+}: {
+  location: LocationSearchResult
+  selected?: boolean
+}) => (
+  <Marker
+    position={[location.latitude, location.longitude]}
+    icon={locationLeafletIcon(selected)}
+  />
+)
```

Comment:
suggestion(non-blocking): consider using `ReactMarker` here

```suggestion

export const LocationMarker = ({
  location,
  selected,
}: {
  location: LocationSearchResult
  selected?: boolean
}) => (
  <ReactMarker
    position={[location.latitude, location.longitude]}
    divIconSettings={{
      iconAnchor: new Leaflet.Point(9, 24),
      iconSize: [24, 18],
      className:
        "c-location-dot-icon" +
        (selected ? " c-location-dot-icon--selected" : ""),
    }}
    icon={<LocationDotIcon />}
  />
)
```

The issue I encountered was that when I have dev tools open and have the `locationDotIcon` svg selected in the DOM. If I pan the map, the selection in the dev tools get's erased, which is kinda annoying, but I also think it means that this `divIcon` is being regenerated every time the surounding state gets updated (such as panning updating the marker posistion in screen space).

When I tried using `ReactMarker` locally for this, I found that this didn't happen anymore, which I believe is because the dom reference for the react portal is preserved across renders.